lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -90,22 +91,42 @@ sub _get_ptero_execute_method {
 
     my $self = shift;
     my $log_dir = shift;
+    my $ptero_lsf_parameters = $self->_get_ptero_lsf_parameters();
+    $ptero_lsf_parameters->{command} = sprintf(
+        'genome ptero wrapper --command-class %s '
+        .'--method execute --log-directory %s',
+        $self->command, $log_dir);
```

Comment:
Does this still work if the desired log directory contains spaces or other characters that a shell might treat specially?  That's my main concern in switching away from an array version.